test/manifolds/projective_space.jl

```diff
@@ -192,42 +192,30 @@ include("../utils.jl")
             @test repr(M) == "ProjectiveSpace(2, ℍ)"
             @test representation_size(M) == (3,)
             @test manifold_dimension(M) == 8
-            @test !is_point(M, Quaternion[1.0 + 0im, 0.0, 0.0, 0.0])
-            @test !is_vector(
-                M,
-                Quaternion[1.0 + 0im, 0.0, 0.0, 0.0],
-                Quaternion[0.0 + 0im, 1.0, 0.0],
-            )
-            @test_throws DomainError is_point(M, Quaternion[1.0, im, 0.0], true)
-            @test !is_point(M, Quaternion[1.0, im, 0.0])
-            @test !is_vector(
-                M,
-                Quaternion[1.0 + 0im, 0.0, 0.0],
-                Quaternion[1.0 + 0im, 0.0, 0.0],
-            )
-            @test !is_vector(
-                M,
-                Quaternion.([1.0 + 0im, 0.0, 0.0]),
-                Quaternion.([-0.5im, 0.0, 0.0]),
-            )
+            @test !is_point(M, Quaternion[1.0, 0.0, 0.0, 0.0])
+            @test !is_vector(M, Quaternion[1.0, 0.0, 0.0, 0.0], Quaternion[0.0, 1.0, 0.0])
+            @test_throws DomainError is_point(M, Quaternion[5.0, 0.0, 0.0], true)
+            @test !is_point(M, Quaternion[5.0, 0.0, 0.0])
+            @test !is_vector(M, Quaternion[1.0, 0.0, 0.0], Quaternion[1.0, 0.0, 0.0])
+            @test !is_vector(M, Quaternion.([1.0, 0.0, 0.0]), Quaternion.([-0.5, 0.0, 0.0]))
```

Comment:
This keeps the original point in every case but uses the more terse `quat` function:
```suggestion
            @test !is_point(M, quat([1.0, 0.0, 0.0, 0.0])
            @test !is_vector(M, quat([1.0, 0.0, 0.0, 0.0]), quat([0.0, 1.0, 0.0]))
            @test_throws DomainError is_point(M, [1.0, quat(0, 1, 0, 0), 0.0], true)
            @test !is_point(M, [1.0, quat(0, 1, 0, 0), 0.0])
            @test !is_vector(M, quat([1.0, 0.0, 0.0]), quat([1.0, 0.0, 0.0]))
            @test !is_vector(M, quat([1.0, 0.0, 0.0]), [quat(0, -0.5, 0, 0), 0.0, 0.0])
```

Reply:
Ah you missed a bracket here ;)